handsontable/src/plugins/hiddenColumns/hiddenColumns.scss

```diff
@@ -18,6 +18,20 @@
   position: relative;
 }
 
+.handsontable[dir="ltr"] th.afterHiddenColumn div.htLeft {
+  margin-left: 10px;
```

Comment:
For consistency, can I suggest using CSS logical properties such as `margin-inline-start,` etc., instead of `[dir]` rules?